### ISSUE TITLE
Ability to register BLS keys independently of validating

### ISSUE THREAD

**Issue by tactical-retreat** [ava-labs/avalanchego#2805]
**Context and scope**

I haven't been able to find a mechanism to register a BLS key other than during the initial staking request. Is that accurate?

If so, it would be nice to have an alternate mechanism to register one for people who have already set up their staking for a long period of time. This would help unblock AWM usage.

**Comment by joshua-kim** [comment]
Yes. I think the only way is to just wait for your validator to expire and re-register it with a BLS key.
